plover/suggestions.py

```diff
@@ -0,0 +1,52 @@
+import collections
+
+Suggestion = collections.namedtuple('Suggestion', 'text steno_list')
+
+
+class Suggestions(object):
+    def __init__(self, dictionary):
+        self.dictionary = dictionary
+
+    def find(self, translation):
+        suggestions = []
+
+        mods = [
+            u'%s',  # Same
+            u'{^%s}',  # Prefix
+            u'{^}%s',
+            u'{^%s^}',  # Infix
+            u'{^}%s{^}',
+            u'{%s^}',  # Suffix
+            u'%s{^}',
+            u'{&%s}',  # Fingerspell
+            u'{#%s}',  # Command
+        ]
+
+        possible_translations = set([translation])
+
+        stripped_translation = translation.strip()
```

Comment:
The problem is entering `\n` in the lookup dialog will still match: `"KPA*": "{^}{-|}"`, maybe we should only strip spaces?
```python
        # Only strip spaces, so patterns with \n or \t are correctly handled.
        stripped_translation = translation.strip(' ')
```